packages/sdk/src/create/discovery.ts

```diff
@@ -5,18 +5,37 @@ import {
   wakuLocalPeerCacheDiscovery,
   wakuPeerExchangeDiscovery
 } from "@waku/discovery";
-import { type Libp2pComponents, PubsubTopic } from "@waku/interfaces";
+import {
+  CreateNodeOptions,
+  type Libp2pComponents,
+  PubsubTopic
+} from "@waku/interfaces";
+
+export const DEFAULT_DISCOVERIES_ENABLED = {
```

Comment:
let's not pollute our export space with this and define what is needed in place - e.g define what discoveries are needed in `test` package as it is separate execution space, otherwise, default values should be default and not needed for export/import to be enabled

Reply:
Ah, good point!